independent-publisher-connectors/IBM Watson Assistant/readme.md

```diff
@@ -0,0 +1,43 @@
+# **IBM Watson Assistant**
```

Comment:
@lucastitus please follow and include all required sections from our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md), and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/Independent%20Publisher/readme.md).